Assets/MixedRealityToolkit.Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -71,6 +84,12 @@ public bool MemoryStatsVisible
             set { memoryStatsVisible = value; }
         }
 
+        private bool ShouldShowMemoryStats => memoryStatsVisible
+#if WINDOWS_UWP
+            && (appCapture == null || !appCapture.IsCapturingVideo)
```

Comment:
Nit: We could remove the checks here since ShouldShowProfiler handles the global visibility of the profiler. I.E. If ShouldShowProfiler is false ShouldShowMemoryStats doesn't matter since the whole window is invisible.

Reply:
Reverted